grantnav/csv_layout.py

```diff
@@ -134,3 +153,35 @@
     "max",
     "min"
 ]
+
+
+def grants_csv_to_dictionary():
+    """ takes grants_csv and turns it into a dictionary of parent/child
+    fields e.g.
+
+    res["Planned Dates"][
```

Comment:
is this meant to mean that dict values are arrays?
```suggestion
    res["Planned Dates"] = [
```

Reply:
yep thanks